src/main/java/com/aws/greengrass/localdebugconsole/SimpleHttpServer.java

```diff
@@ -221,7 +221,7 @@ public void startup() throws InterruptedException {
     static String fingerprintCert(X509Certificate cert)
             throws NoSuchAlgorithmException, CertificateEncodingException {
         StringBuilder sb = new StringBuilder();
-        for (byte b : MessageDigest.getInstance("SHA-256").digest(cert.getEncoded())) {
+        for (byte b : MessageDigest.getInstance("SHA-1").digest(cert.getEncoded())) {
```

Comment:
SHA-1 is broken. Do we really want to do this?

Reply:
Not entirely true that it is broken, but yes we do want to use it. Windows doesn't provide the sha-256 hash for viewing. So sha-1 is better than no hash at all.

Reply:
More recent attacks have improved, but I definitely recognize that this is likely not a worthwhile target for a well funded attacker. Regardless, we should still promote security best practices here. Agree that SHA-1 is better than no hash at all, for Windows users. 

Can we provide both?